api/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>jumpstart-backend</artifactId>
+    <groupId>com.codeforcommunity</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>api</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.codeforcommunity</groupId>
+      <artifactId>common</artifactId>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
It might be worth adding comments to the pom.xml files (maybe just the root one) that explain the purpose and link to a code example for each dependency.

Reply:
Working on that now.